azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md

```diff
@@ -0,0 +1,106 @@
+# DPS CERTIFICATE MANAGEMENT
+
+In a nutshell, IoT Device can request a client certificate from a CA through DPS. 
+* Connect DPS to the private CA hosted by one of our CA partners. 
+* IoT device sends a CSR to DPS
+* DPS forwards it to the CA for signing and returns an X.509 client certificate to device. 
+* Device uses certificate to authenticate with IoT Hub.
+
+## Steps for making it work in preview (api-version=2021-11-01-preview)
+
+### Prerequisite Steps for DPS , enrollment and CA
+
+These steps must be done for any scenario inside DPS Cert Management.
+
+* A new DPS instance must created in __West Central US__.
+* Create an Enrollment Group or Individual Enrollment in this DPS instance.
+    * Choose to use symmetric key, TPM key or X.509 for this enrollment.
+    * Link an IoT Hub to the enrollment.
+* Create an internal-use test account with one of Microsoft CA partners by emailing _iotcerts@microsoft.com_
```

Comment:
This says "internal only". Is this step valid for people outside of Microsoft?

Reply:
No not right now. This document will not be valid once this entire feature goes to public (service + device). But half of these things are not really written anywhere and I had to spend some time discovering some of the commands. So it is helpful now.